provider/marathon/marathon_test.go

```diff
@@ -934,440 +699,563 @@ func TestMarathonGetProtocol(t *testing.T) {
 		c := c
 		t.Run(c.desc, func(t *testing.T) {
 			t.Parallel()
+			provider := &Provider{}
 			actual := provider.getProtocol(c.application)
 			if actual != c.expected {
-				t.Errorf("got protocol '%s', want '%s'", actual, c.expected)
+				t.Errorf("actual %q, expected %q", actual, c.expected)
 			}
 		})
 	}
 }
 
-func TestMarathonGetPassHostHeader(t *testing.T) {
-	provider := &Provider{}
-
-	applications := []struct {
+func TestMarathonGetSticky(t *testing.T) {
+	cases := []struct {
+		desc        string
 		application marathon.Application
 		expected    string
 	}{
 		{
-			application: marathon.Application{
-				Labels: &map[string]string{}},
+			desc:        "label missing",
+			application: application(),
```

Comment:
@vdemeester's out-of-band feedback is that he more often than not drops the `with` prefix in his builders. :-)

So I just added the `create` prefix. Not sure if it violates fluentd builder syntaxes. I did this a lot in Java, but maybe that's a sin of the past. ;)